Files already downloaded and verified
Files already downloaded and verified
Epoch 1 - train loss: 2.382336:   1%| | 1/98 [00:06<11:18,
Traceback (most recent call last):
  File "/home/sungwoopark/dl-experiment/main.py", line 73, in <module>
    train(model, train_loader, optimizer, scheduler)
  File "/home/sungwoopark/dl-experiment/main.py", line 18, in train
    output = model(image)
  File "/home/sungwoopark/anaconda3/envs/main/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/sungwoopark/dl-experiment/model.py", line 126, in forward
    x = F.relu(self.conv_bn11(x))
  File "/home/sungwoopark/anaconda3/envs/main/lib/python3.9/site-packages/torch/nn/functional.py", line 1457, in relu
    result = torch.relu(input)
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 128.00 MiB (GPU 0; 23.70 GiB total capacity; 5.58 GiB already allocated; 71.56 MiB free; 5.65 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF